Dockerfile

```diff
@@ -1,6 +1,11 @@
 # Stage 1 - Install dependencies and build the app
 # This matches the flutter version on our CI/CD pipeline on Github
-FROM cirrusci/flutter:3.7.5 AS build-env
+FROM debian AS build-env
+
+RUN apt-get update && \
+    apt-get install -y curl git wget unzip
+RUN git clone --depth=1 --branch=3.7.5 https://github.com/flutter/flutter.git /usr/local/flutter
+ENV PATH="/usr/local/flutter/bin:/usr/local/flutter/bin/cache/dart-sdk/bin:${PATH}"
```

Comment:
IMO, no need to change it, we not run it in M1 in production. Only change it for local development